### PR TITLE
Uploading BT Environment for jackson-core

### PR DESCRIPTION
**Pull request by Aditi8007**

### CODE REVIEW

File: jackson-core/0.0.1/besman-jackson-core-BT-env-config.yaml [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+# The variables that start with BESMAN_ are converted to environment vars.
+# If you wish to add any other vars that should be used globally, add the var using the below format.
+# BESMAN_<var name>: <value>
+# If you are not using any particular value, remove it or comment it(#).
+#*** - These variables should not be removed, nor left empty.
+# BESMAN_ORG - used to mention where you should clone the repo from, default value is Be-Secure
+BESMAN_ORG: Be-Secure #***
+
+# BESMAN_ARTIFACT_TYPE - project/ml model/training dataset
+BESMAN_ARTIFACT_TYPE: # project/ml model/training dataset #***
```
Comment (asa1997) [inline jackson-core/0.0.1/besman-jackson-core-BT-env-config.yaml:13]: Should be project

File: jackson-core/0.0.1/besman-jackson-core-BT-env-config.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+# BESMAN_ARTIFACT_NAME - name of the artifact under assessment.
+BESMAN_ARTIFACT_NAME: jackson-core #***
+
+# BESMAN_ARTIFACT_VERSION - version of the artifact under assessment.
+BESMAN_ARTIFACT_VERSION: #Enter the version of the artifact here. #***
```
Comment (asa1997) [inline jackson-core/0.0.1/besman-jackson-core-BT-env-config.yaml:19]: Should add the version under assessment.

**Review by asa1997 — CHANGES_REQUESTED** [review]
@Aditi8007 Please address the requested changes.

**Comment by Aditi8007** [comment]
Now please check

File: list.txt [hunk 3]
```diff
@@ -12,3 +12,5 @@ Be-Secure/besecure-ce-env-repo/lettuce-BT-env,0.0.1
 
 Be-Secure/besecure-ce-env-repo/opencti-RT-env,0.0.1
 
+Be-Secure/besecure-ce-env-repo/jackson-core-BT-env,0.0.1
+Be-Secure/besecure-ce-env-repo/druid-BT-env,0.0.1
```
Comment (asa1997) [inline list.txt:16]: Please remove this

**Review by asa1997 — CHANGES_REQUESTED** [review]
@Aditi8007 You have an additional entry into the list file

**Review by asa1997 — APPROVED** [review]
@Aditi8007 LGTM.

**Pull request merged**